### ISSUE TITLE
Drag and Drop to Mail app doesnt work

### ISSUE THREAD

**Issue by sguergachi** [files-community/Files#2924]
**Describe the bug**

I tried to drag a PDF from the FIles app to the Microsoft Mail app compose pane. I am able to drag a file from the Native File Explorer into the compose pane of an the Mail app.

**To Reproduce**
1. Go to Files app and find any file to attach to the Mail app, like a PDF
2. Start a new email in Mail app to open compose pane in the app
3. Drag file from Files app to Mail app compose pane
4. Nothing happens

**Expected behavior**

Mail app to accept drag and drop file as expected in all apps

**Screenshots**

**Desktop (please complete the following information):**
 - OS Version:  Windows 10 19042.685

 - App version: 1.0.0.0

**Additional context**
Add any other context about the problem here. Does this problem occur again after restarting the app?

**Log file**
[debug.txt](https://github.com/files-community/Files/files/5799198/debug.txt)

**Comment by dckface** [comment]
This is reported for v1.0, but it seems to be happening as well in v1.1 (Win 10 Pro, 20H2).

**Comment by RealCyGuy** [comment]
Many apps don't work when I try dragging them in. Chrome seems to work, I can attach a file here.

**Comment by sguergachi** [comment]
In general drag and drop seems buggy. Sometimes I try to drag something and it doesn't pickup. So I have to select it first then drag.

**Comment by RealCyGuy** [comment]
Yes, buggy. But I think that's intended, you have to click first.

**Comment by sguergachi** [comment]
That doesnt feel intuitive considering native Windows doesnt require selecting first, and selecting first is cumbersome.
